src/app/features/vendor/vendor-view/vendor-view.component.html

```diff
@@ -0,0 +1,77 @@
+<p-confirmDialog
+  header="Confirmation"
+  icon="pi pi-exclamation-triangle"
+></p-confirmDialog>
+<div class="container">
+  <div class="nav"></div>
+  <div class="header">
+    <h1>
+      {{ vendor?.name }}
+      <span *ngIf="!vendor?.isApproved" class="status-label pending">
+        PENDING
+      </span>
+    </h1>
+    <div class="buttons">
+      <button
+        pButton
+        id="editProfileButton"
```

Comment:
update the id to edit-profile-btn. Use same format in other buttons also.

Reply:
Ok noted